Sources/Argo/Extensions/NSNumber.swift

```diff
@@ -2,6 +2,6 @@ import Foundation
 
 extension NSNumber {
   var isBool: Bool {
-    return CFBooleanGetTypeID() == CFGetTypeID(self)
+    return type(of: self) == type(of: NSNumber(value: true))
```

Comment:
do we need to construct `NSNumber` to check if the type is `Bool`? could we simply compare it to `Bool` or `type(of: true)`?